cci/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ SET_SOURCE_FILES_PROPERTIES(
 
 add_library(cascci SHARED ${CASCCI_SOURCES})
 set_target_properties(cascci PROPERTIES SOVERSION "${CUBRID_MAJOR_VERSION}.${CUBRID_MINOR_VERSION}")
-set_target_properties(cascci PROPERTIES PUBLIC_HEADER "${CCI_DIR}/cas_cci.h;${BROKER_DIR}/cas_error.h")
+set_target_properties(cascci PROPERTIES PUBLIC_HEADER "${CCI_DIR}/cas_cci.h;${CCI_DIR}/cas_cci_common.h;${BROKER_DIR}/cas_error.h")
```

Comment:
please revert adding "cas_cci_common.h". I don't want to suffer from a distribution/compatibility issue of CCI driver. 
Since there's no mixed usages of compat and cci api and we will not modify the block for backward compatibility, duplications will not matter. 
I know it is ugly and hope you don't mind it.